docs/howto/features/exam.md

```diff
@@ -0,0 +1,38 @@
+# Set up a hub for an exam
+
+We provide support for using a JupyterHub in a *controlled* physical
+space for exams. This is an extra paid feature where we charge hourly
+for responsiveness.
+
+This page documents what we do to prep, based on our prior experiences.
+
+1. Make sure the exact dates and times of the exam are checked well in
+   advance, and we have enough engineering coverage during this time period.
+   Engineers should also *test* their access to the infrastructure and the
```

Comment:
Can we be more specific here with a checklist maybe

- [ ] Can access and login to the hub admin page (see https://infrastructure.2i2c.org/en/latest/reference/hubs.html for links)
- [ ] Can access and login to the cluster grafana
- [ ] Can access and login to the cloud console 
- [ ] Test access to Logs Explorer for container logs if on GCP
- [ ] Test `deployer use-cluster-credentials $CLUSTER` and then `kubectl get pods -A` work